### PR TITLE
Fixing typos as we go

### PR DESCRIPTION
**Pull request by tarnas14**
Fixing typos and skipping test initially for better console experience

### CODE REVIEW

**Comment by tarnas14** [comment]
maybe `it.skip` would be better instead of throw(new Error) ?

File: src/01/02.db.test.js [hunk 1]
```diff
@@ -4,7 +4,8 @@
 // 🦖 Your job is to replace the comments with actual code, have fun!
 // 💡 Remember to unskip!
 describe.skip('db', () => {
-  it(`should create empty collection when it doesn't exist`, () => {
+  // 💡 Remember to unskip!
+  it.skip(`should create empty collection when it doesn't exist`, () => {
```
Comment (kjendrzyca) [inline src/01/02.db.test.js]: I think better combination would be:
```
describe.skip(...
  it.only(...
```

by default

Comment (tarnas14) [reply src/01/02.db.test.js]: changed to setting first test under describe to `only`

Comment (kjendrzyca) [reply src/01/02.db.test.js]: :muscle:

**Comment by kjendrzyca** [comment]
@all-contributors please add @tarnas14 for code and maintenance

**Pull request merged**